plat/common/include/x86/cfi.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
I think you should move this header to x86 arch:
`arch/x86/x86_64/include/uk/asm`
You should then also update the commit subject prefix to `arch/x86`.